src/Pages/2DPrinting/2DPrinting.js

```diff
@@ -49,15 +55,90 @@ export default function Printing(props) {
     props.user.pagesPrinted
   );
   const [previewDisplay, setPreviewDisplay] = useState('');
+  const [loadPreview, setLoadPreview] = useState(true);
+  const [statusModal, setStatusModal] = useState(false);
+  const [printStatus, setPrintStatus] = useState('');
+
   const previewLabels = {
```

Comment:
you might as well throw all these `const` definitions into `2DComponents.js`

Reply:
But they're used in the loading mechanism/printing mechanism of the actual page and are modified by other methods in the same function...

Reply:
ok nvm